azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/EventHubClient.java

```diff
@@ -102,10 +117,28 @@ public static EventHubClient createFromConnectionStringSync(final String connect
 	public static CompletableFuture<EventHubClient> createFromConnectionString(final String connectionString)
 			throws ServiceBusException, IOException
 	{
+		return createFromConnectionString(connectionString, null);
```

Comment:
>null [](start = 54, length = 4)

pass connStr.getRetryPolicy() - if this constructor (without retryPolicy) was used (or) deprecate RetryPolicy from ConnectionString. Otherwise, this could confuse users.